internal/http/services/owncloud/ocdav/put.go

```diff
@@ -223,19 +223,19 @@ func (s *svc) handlePut(ctx context.Context, w http.ResponseWriter, r *http.Requ
 		Opaque: &typespb.Opaque{Map: opaqueMap},
 	}
 
-       dirPart, filePart := filepath.Split(ref.Path)
-       dirParts := strings.Split(dirPart, "/")
-       folder := dirParts[len(dirParts)-1]
-
-       trg := &trigger.Trigger{
-               Ref: "any",
-               TemplateData: map[string]interface{}{
-                       "path":     ref.Path,
-                       "folder":   folder,
-                       "fileName": filePart,
-               },
-       }
-       s.notificationHelper.TriggerNotification(trg)
+	dirPart, filePart := filepath.Split(ref.Path)
+	dirParts := strings.Split(dirPart, "/")
+	folder := dirParts[len(dirParts)-1]
+
+	trg := &trigger.Trigger{
+		Ref: "any",
+		TemplateData: map[string]interface{}{
+			"path":     ref.Path,
+			"folder":   folder,
+			"fileName": filePart,
+		},
+	}
+	s.notificationHelper.TriggerNotification(trg)
```

Comment:
This is part of our change to generate notifications on all uploads, so you can remove it.